ci/deps.sh

```diff
@@ -34,40 +35,72 @@ echo "MAKEFLAGS: ${MAKEFLAGS}"
 mkdir -p /tmp
 cd /tmp
 
+gh_retrieve_wget(){
+    ORG=$1
+    REPO=$2
+    TAG=$3
+
+    echo "Downloading $ORG $REPO $TAG"
+    
+
+
+    wget $WGET_FLAGS https://github.com/$ORG/$REPO/archive/refs/tags/$TAG.tar.gz && \
```

Comment:
For now, there are only references to github repositories (except for AVRO, but you moved it to use the official repo in github). That might not always be the case, and then this function should be rewritten / write another one. I would pass the entire URL.

Reply:
Hi Marc,
Thanks for the idea, I'll try to have a more generic approach in case we have something that is not hosted on Github.